thirdparty/libgc/amalgamation.txt

```diff
@@ -0,0 +1,7 @@
+The libgc source is distributed here as an amalgamation (https://sqlite.org/amalgamation.html).
+This means that, rather than mirroring the entire bdwgc repo here,
+[this script](https://gist.github.com/spaceface777/34d25420f2dc4953fb7864f44a211105) was used
+to bundle all local includes together into a single C file, which is much easier to handle.
+Furthermore, the script above was also used to minify (i.e. remove comments and whitespace in)
```

Comment:
I think the source [URL](https://github.com/ivmai/bdwgc) should be mentioned somewhere. Also which version (git commit Id or release tag) was used to generate the amalgamation. (I'd prefer a released version like v8.0.4)

Reply:
I think the script should be inside the V repo, perhaps in cmd/tools ?